examples/00-EDB/10_GDS_workflow.py

```diff
@@ -16,39 +15,40 @@
 from pyaedt.edb_core.edb_data.control_file import ControlFile
 
 ###############################################################################
-# Download file
-# ~~~~~~~~~~~~~
-# Download the AEDB file and copy it in the temporary folder.
-temppath = tempfile.gettempdir()
-local_path = pyaedt.downloads.download_file('gds')
-c_file_in = os.path.join(
-    local_path, "sky130_fictitious_dtc_example_control_no_map.xml"
-)
-c_map = os.path.join(local_path, "dummy_layermap.map")
-gds_in = os.path.join(local_path, "sky130_fictitious_dtc_example.gds")
-gds_out = os.path.join(temppath, "example.gds")
+# ## Fetch Example Data
+#
+# Download the EDB folder and copy it to a temporary folder.
+# The following files are used in this example:
+# - _sky130_fictious_dtc_exmple_contol_no_map.xml_
+#   defines physical information such
+#   as material properties, stackup layers, and boundary conditions.
+# - _dummy_layermap.map_
+#   maps properties to stackup layers.
+
+temp_dir = tempfile.TemporaryDirectory(suffix=".ansys")
+control_fn = "sky130_fictitious_dtc_example_control_no_map.xml"
+gds_fn = "sky130_fictitious_dtc_example.gds"
+layer_map = "dummy_layermap.map"
+
+local_path = pyaedt.downloads.download_file('gds', destination=temp_dir.name)
+c_file_in = os.path.join(local_path, control_fn)
+c_map = os.path.join(local_path, layer_map)
+gds_in = os.path.join(local_path, gds_fn)
+gds_out = os.path.join(temppath, "gds_out.gds")
 shutil.copy2(gds_in,gds_out )
-###############################################################################
-# Control file
-# ~~~~~~~~~~~~
-# A Control file is an xml file which purpose if to provide additional
-# information during import phase. It can include, materials, stackup, setup, boundaries and settings.
-# In this example we will import an existing xml, integrate it with a layer mapping file of gds
-# and then adding setup and boundaries.
-
+""
 c = ControlFile(c_file_in, layer_map=c_map)
 
-
 ###############################################################################
-# Simulation setup
-# ~~~~~~~~~~~~~~~~
+# ## Simulation setup
+#
 # Here we setup simulation with HFSS and add a frequency sweep.
```

Comment:
```suggestion
# This code sets up a simulation with HFSS and adds a frequency sweep.
```
Don't use "we" but you or rewrite with a new subject performing the action as I did here.